### ISSUE TITLE
Specify project settings in Cartfile.private

### ISSUE THREAD

**Issue by jspahrsummers** [Carthage/Carthage#262]
Basically, everything configurable on the `Project` type should have a corresponding `Cartfile.private` flag. I don't think this information should go into the `Cartfile`, because the settings shouldn't affect parent projects.

Any extensions we make should conform to the [OGDL](http://ogdl.org) spec.

**Comment by robrix** [comment]
Regarding [your tweet](https://twitter.com/jspahrsummers/status/551635102904487938), if you want to do this with parser combinators, https://github.com/robrix/Madness is parser combinators.

**Comment by paulyoung** [comment]
What about flags for single platforms or `--no-build`?

I took a quick look at `Project` (on my phone) but couldn't see those.

**Comment by jspahrsummers** [comment]
`--no-build` is out of scope for the time being. Platform flags are part of #127.

**Comment by jspahrsummers** [comment]
This is more-or-less blocked by https://github.com/Carthage/ogdl-swift/pull/1.
